docs/src/modules/utils/find.js

```diff
@@ -118,6 +118,15 @@ function findPages(
     });
   });
 
+  // sort by pathnames without '-' so that e.g. card comes before card-action
+  pages.sort((a, b) => {
+    const pathnameA = a.pathname.replace(/-/g, '');
```

Comment:
Ideally, we should have been ignoring the `.js` extension:
```js
> ['card-action', 'card'].sort()
["card", "card-action"]
```

But it's working, so let's 🚢 .